docs(api): rephrase description (#4371)
webpack/webpack.js.org: src/content/api/cli.md

__Build source using a configuration file__

> Specify a different [configuration](/configuration) file to pick up. Use this if you want to specify something different from `webpack.config.js`, which is one of the defaults.

Reviewer: ```suggestion
Specify a different [configuration](/configuration) file other than `webpack.config.js`, which is one of the defaults.
```
